Modules/Microsoft365DSC/DSCResources/MSFT_AADConnectorGroupApplicationProxy/MSFT_AADConnectorGroupApplicationProxy.psm1

```diff
@@ -0,0 +1,455 @@
+function Get-TargetResource
+{
+    [CmdletBinding()]
+    [OutputType([System.Collections.Hashtable])]
+    param
+    (
+        [Parameter()]
+        [System.String]
+        $Name,
+
+        [Parameter()]
+        [ValidateSet('nam','eur','aus','asia','ind','unknownFutureValue')]
+        [System.String]
+        $Region,
+
+        [Parameter(Mandatory = $true)]
+        [System.String]
+        $Id,
```

Comment:
Should name be the Key to ensure we can replicate across tenants, or are ID immutable across tenants?

Reply:
Done